### ISSUE TITLE
usage: !timer enable/disable twitter

### ISSUE THREAD

**Issue by Libeylul** [mohamed-tayeh/Minimal-Pomo-Timer#41]
The bot literally says that when I try to use the timer, and the message makes no sense to me. I've never used twitter. I tried to disable it by typing !timer disable or !timer disable twitter but it literally answer the same thing over and over : usage: !timer enable/disable twitter

**Comment by Moh-inc** [comment]
Do you have another bot in your stream such as streamelements/streamlabs/nightbot/fossabot?  That message is most likely coming from those bots

**Comment by Libeylul** [comment]
> Do you have another bot in your stream such as streamelements/streamlabs/nightbot/fossabot? That message is most likely coming from those bots

I do have streamelements, but command is turned off and timer do not go down anyway. I don't think I have any other bot as my stream are usually simple with nothing fancy.

**Comment by Moh-inc** [comment]
@Libeylul 

- Are you sure you have setup pomo bot correctly? Because there is no link between the pomo bot and twitter
- Can you try removing streamelements from your channel and see if the timer command gets triggered?

If these two don't work, we can continue the conversation offline on discord. https://discord.gg/RSVThTwK6K

**Comment by Libeylul** [comment]
I will do so when I get back home early next week, thank you for your time.

**Comment by Libeylul** [comment]
![image](https://github.com/user-attachments/assets/0851bea3-25fd-4fab-80a4-612fb3fb8fcf) So yea, timer do not go down when removing streamelements but the message does not disappear


> @Libeylul
> 
> * Are you sure you have setup pomo bot correctly? Because there is no link between the pomo bot and twitter
> * Can you try removing streamelements from your channel and see if the timer command gets triggered?
> 
> If these two don't work, we can continue the conversation offline on discord. https://discord.gg/RSVThTwK6K

**Comment by Sweedy2705** [comment]
I have the same problem

**Comment by Moh-inc** [comment]
@Libeylul are you still facing the same problem?

**Comment by TheRealMsNoName** [comment]
> @Libeylul are you still facing the same problem?

Hello, I experienced the same issue two days ago, after deleting everything and reinstalling and eliminated Stream Elements, the Pomodoro Timer is still not responding to my command thru the chat.

**Comment by Libeylul** [comment]
> @Libeylul are you still facing the same problem?

Yes I am still facing this issue. Despite trying again, and following every step by step, it still doesn't work when it did a long time ago.

**Comment by Moh-inc** [comment]
@Libeylul @TheRealMsNoName Can you please join my discord so that we can discuss it in more detail? I think there are some steps that might be unclear in the setup.



https://discord.gg/Jbta7C7HEY

**Comment by TheRealMsNoName** [comment]
> @Libeylul @TheRealMsNoName Can you please join my discord so that we can discuss it in more detail? I think there are some steps that might be unclear in the setup.
> 
> https://discord.gg/Jbta7C7HEY

Sure....I will be there

**Comment by TheRealMsNoName** [comment]
> @Libeylul @TheRealMsNoName Can you please join my discord so that we can discuss it in more detail? I think there are some steps that might be unclear in the setup.
> 
> https://discord.gg/Jbta7C7HEY

I joined, in which of the session should I ask the question? I am Ms. No Name in Discord.  Thank you
